chia/wallet/wallet_nft_store.py

```diff
@@ -157,6 +157,19 @@ async def count(self, wallet_id: Optional[uint32] = None, did_id: Optional[bytes
                 return int(count_row[0])
         return -1
 
+    async def is_empty(self, wallet_id: Optional[uint32] = None) -> bool:
+        sql = "SELECT 1 FROM users_nfts WHERE removed_height is NULL"
```

Comment:
`select exists(SELECT 1 FROM users_nfts WHERE removed_height is NULL)` returns either 0 or 1, unclear if it's any better but it is more typical

Reply:
Thanks, I was trying to make it as simple as possible and to avoid nesting.